dev/com.ibm.ws.microprofile.rest.client.FT_fat/fat/src/com/ibm/ws/microprofile/rest/client/fat/RetryTest.java

```diff
@@ -49,7 +53,11 @@
 
     @BeforeClass
     public static void setUp() throws Exception {
-        ShrinkHelper.defaultDropinApp(server, appName, SERVER_NAME);
+        WebArchive war = ShrinkHelper.buildDefaultApp(appName, SERVER_NAME);
+        StringAsset mpConfig = new StringAsset(RetryClient.class.getName() + "/mp-rest/uri=http://localhost:"
+                        + System.getProperty("bvt.prop.HTTP_default", "8010") + "/retryApp");
+        war.addAsWebInfResource(mpConfig, "classes/META-INF/microprofile-config.properties");
+        ShrinkHelper.exportDropinAppToServer(server, war);
```

Comment:
recommend adding `DeployOptions.SERVER_ONLY` otherwise there can be contamination between repeats, particularly when mixing EE8 and EE9